imdigitalashish.md

```diff
@@ -0,0 +1,46 @@
+---
+name: "@imdigitalashish"
+project: "Zenith"
+---
+
+
+# Zenith
+
+## Summary
+
+It is a home automation tool that resembles IRON man Jarvis Assistant. I will create this from groud up including all codes and hook
+it up with hardware to automate my common tasks and open source it so everyone can use it and fullfill their IRON man dream !
+
+## Plan
+
+I will use Raspberry Pi 4 as main controller of the board running RUST (for key performance areas) and Python (for AI / ML). With that
+I proceed by connecting to hardware by wifi or wires or bluetooth to control and get data and make variety of features such as intruder detection (servo motor), turn on lights, integrate it with ChatGPT and my own chat model to communicate, make a beautifull GUI'S, create a server to communicate over call through Azure and control my room from anywhere in the world and to store stuff by just saying "JARVIS SAVE THIS FILE" to a harddrive using raspberry PI ( just like airdrop ) and incrementally add features time to time.
+
+Features that I will be making:
+
+1) Apple Airdrop feature with Harddrive and Raspberry PI
+2) Face Recognition Door lock using Servo Motor ( Here Raspberry PI send commands to Arduino with SSE Events ) and Potentiometer
```

Comment:
so would your raspberry pi be mounted on your door?

Reply:
Again thank you for this question.... !! I did some R&D  So,

I will use raspberry PI pico (because all other components will be controlled over wifi).

I will use Main RASPBERRY PI over wifi to send commands to raspberry PI pico to control the servo motor. I also have a reference on how I will do that 

[Youtube link with timestamp](https://youtu.be/AK8UYh7pMGM?t=466)

Reply:
ahh great, can you please elaborate on this under plan within your file?

Reply:
Thank you, I have added this change also !